### PR TITLE
Added import test to pymisc-utils

### PR DESCRIPTION
**Pull request by RickGelhausen**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pymisc-utils/build.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 $PYTHON setup.py install
-cp misc/*.py ${PREFIX}/bin/
+find misc -name "*.py" -exec cp {} ${PREFIX}/bin/ \;
```
Comment (johanneskoester) [inline recipes/pymisc-utils/build.sh]: What was wrong with the cp before?

Comment (RickGelhausen) [reply recipes/pymisc-utils/build.sh]: My bad, I forgot to change that back. I tested whether it caused an error I fixed with the patch.
I will change it back.

Comment (johanneskoester) [reply recipes/pymisc-utils/build.sh]: No problem, thanks!

**Comment by bgruening** [comment]
Thanks @RickGelhausen!

**Pull request merged**